docs/tools/vdb_table/data/clickhouse.json

```diff
@@ -113,9 +113,9 @@
     "comment": ""
   },
   "in_process": {
-    "support": "none",
+    "support": "full",
```

Comment:
```suggestion
    "support": "partial",
```

Reply:
Because chDB isn't maintained by the parent company (ClickHouse) and is potentially offered by a third-party in the long term, it makes sense to not state this as `full` because in essence, we'd be pointing users to a different tool that has the in-process capability.